spring-cloud-kubernetes-commons/pom.xml

```diff
@@ -50,13 +50,29 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>org.mockito</groupId>
+					<artifactId>mockito-core</artifactId>
+				</exclusion>
+			</exclusions>
 			<scope>test</scope>
 		</dependency>
 		<dependency>
 			<groupId>org.junit.vintage</groupId>
 			<artifactId>junit-vintage-engine</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-inline</artifactId>
+		</dependency>
+		<dependency>
```

Comment:
needed because of `TestPropertySourceUtils::addInlinedPropertiesToEnvironment` in `AbstractKubernetesProfileEnvironmentPostProcessorTest`. This is how an `EnvironmentPostProcessor` is tested